pkgs/applications/video/obs-studio/wlrobs.nix

```diff
@@ -0,0 +1,36 @@
+# We don't have a wrapper which can supply obs-studio plugins so you have to
+# somewhat manually install this:
+
+# nix-env -f . -iA wlrobs
+# mkdir -p ~/.config/obs-studio/plugins
+# ln -s ~/.nix-profile/share/obs/obs-plugins/wlrobs ~/.config/obs-studio/plugins/
```

Comment:
Suggestion: some packages put this kind of useful information in `meta.longDescription` so that it is displayed on the online package search page, for example.

Reply:
SGTM, I've transferred this information into `longDescription`. It probably would not hurt to do the same for the other OBS plugin later on.